bigchaindb/common/transaction.py

```diff
@@ -757,6 +767,10 @@ def gen_public_key(private_key):
         tx_serialized = Transaction._to_str(tx_dict)
         for i, input_ in enumerate(self.inputs):
             self.inputs[i] = self._sign_input(input_, tx_serialized, key_pairs)
+
+        if and_hash:
```

Comment:
Can you write a test case where we set `and_hash` to False?

Reply:
I removed the `and_hash` keyword argument. For now fulfilling (signing) will implicitly hash. This is not ideal and can be tackled in #1911.